### PR TITLE
Sane error for missing pause bundle

### PR DESCRIPTION
**Pull request by jodh-intel**

### CODE REVIEW

**Comment by dlespiau** [comment]
and +2.3% coverage, what else can we ask?

lgtm

File: main_test.go [hunk 1]
```diff
@@ -49,3 +49,7 @@ func TestMain(m *testing.M) {
 
 	os.Exit(ret)
 }
+
+func createEmptyFile(path string) (err error) {
```
Comment (devimc) [inline main_test.go]: Q. Could this function be part of a different file? something like common.go or lib.go?

Comment (jodh-intel) [inline main_test.go]: I guess it could. Whether we want to do that now or maybe later when we have a few other functions to put in the file I don't know?

Comment (dlespiau) [inline main_test.go]: Generic files like "common.go", "lib.go", "utils.go" always end up with a mixed bag of "things". Could maaaybe be fileutils_test.go, it's more definite already.

Comment (dlespiau) [inline main_test.go]: Later is certainly more than fine.

**Comment by devimc** [comment]
lgtm

**Pull request merged**